Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM rustembedded/cross:aarch64-linux-android AS aarch64-linux-android
+
+ENV TZ=Etc/UTC
+RUN \
+	DEBIAN_FRONTEND=noninteractive \
+	ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone \
+	&& apt-get update && apt-get -y install lsb-release sudo python git \
+	&& git clone https://github.com/denoland/chromium_build /chromium_build \
```

Comment:
So `chromium_build` is actually a submodule of this repo (it gets cloned into the `build` directory).
Cloning the default branch does not seem to be a good idea, because it's essentially at a random state. I happen to have sync'd it with upstream two weeks ago, but that's merely a coincidence (also it does not include the patches we're floating).

You should somehow find a way to use the same commit as the `build` submodule, so this doesn't break randomly.

Reply:
👌

Reply:
I fixed that by copying the `build` directory instead of cloning it
It will only break if user have changed something itself locally